src/Rules/Classes/EnumSanityRule.php

```diff
@@ -70,15 +69,6 @@ public function processNode(Node $node, Scope $scope): array
 				}
 			}
 
-			// serialize/unserialize is not considered a magic method by PhpParser
```

Comment:
no longer requires, as php-parser itself handles this methods as magic.

see changelog
> Added __serialize and __unserialize to magic method list.